### ISSUE TITLE
DX9 support not functioning

### ISSUE THREAD

**Issue by DieuDeGlace** [mj-ehsan/NiceGuy-Shaders#19]
NGlighting fails to compile with error code X4509, "texture limit reached at 16 slots" which according to crosire means that it doesnt support dx9. which doesnt make much sense as it has a function to work with dx9

**Comment by DieuDeGlace** [comment]
yeah says "C:\Program Files (x86)\Steam\steamapps\common\Test Drive Unlimited 2\Shader@0x54E09E60(94,11-31): error X4509: maximum sampler register index exceeded, target has 16 slots, manual bind to slot s17 failed"

**Comment by Mike4080** [comment]
> NGlighting fails to compile with error code X4509, "texture limit reached at 16 slots" which according to crosire means that it doesnt support dx9. which doesnt make much sense as it has a function to work with dx9

Use [DXVK](https://github.com/doitsujin/dxvk/releases/) (specifically the d3d9.dll for Win32 or Win64 depending on the game) and reinstall ReShade for the game, this time choosing Vulkan.
